### PR TITLE
Additional changes to MTA 5.1.1 release notes

### PR DESCRIPTION
**Pull request by RichardHoch**
Changes accidently left out of previous PR and additional changes requested by Phil C in https://issues.redhat.com/browse/WINDUP-2944.

Preview: http://file.emea.redhat.com/rhoch/19012021/rn2/html-single/#rn-resolved-issues_release-notes

### CODE REVIEW

**Comment by RichardHoch** [comment]
@apinnick : please review.

File: docs/topics_5/rn-resolved-issues.adoc [hunk 1]
```diff
@@ -17,13 +17,13 @@ For a full list of all issues resolved in this release, see the list of link:htt
 |MTA reports load slowly, do not include Target Runtime labels on Application list, and do not have working feedback controls.
 
 |link:https://issues.redhat.com/browse/WINDUP-2333[WINDUP-2333]
-|The MTA web console installed on OpenShift by means of a template cannot access the reports using HTTPS.
+|The {WeBname} installed on OpenShift 4.4 and earlier versions cannot access the generated reports over an HTTPS connection.
```
Comment (apinnick) [inline docs/topics_5/rn-resolved-issues.adoc:20]: Fix these attributes. WebName

**Comment by RichardHoch** [comment]
@apinnick: please review.

File: docs/topics_5/rn-known-issues.adoc [hunk 2]
```diff
@@ -3,9 +3,9 @@
 [id='rn-known-issues_{context}']
 = Known issues
 
-At the time of release, the following known issues are identified as major issues in {ProductName} {DocInfoProductNumber}.
+At the time of release, the following known issues have been identified as major issues worth highlighting.
 
-// For a full list of all known issues, see the link:https://issues.redhat.com/issues/?filter=12356604[open {ProductShortName} issues] in Jira.
+For a full list of all known issues, see the list of link:https://issues.redhat.com/browse/WINDUPRULE-607?filter=12357834[{DocInfoProductNumber} known issues] in Jira.
```
Comment (apinnick) [inline docs/topics_5/rn-known-issues.adoc:8]: I thought Phil wanted the text to read "5.1.1" issues. If so, change this attribute.

File: docs/topics_5/rn-resolved-issues.adoc [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 At the time of the release the following resolved issues have been identified as major issues worth highlighting.
 
-For a full list of all issues resolved in this release, see the list of link:https://issues.redhat.com/browse/WINDUP-2759?filter=12357835[{ProductVersion} resolved issues] in Jira.
+For a full list of all issues resolved in this release, see the list of link:https://issues.redhat.com/browse/WINDUP-2759?filter=12357835[{DocInfoProductNumber} resolved issues] in Jira.
```
Comment (apinnick) [inline docs/topics_5/rn-resolved-issues.adoc:8]: Again, link points to 5.1.1 resolved issues. Change the attribute.

File: docs/topics_5/rn-resolved-issues.adoc [hunk 4]
```diff
@@ -5,7 +5,7 @@
 
 At the time of the release the following resolved issues have been identified as major issues worth highlighting.
 
-For a full list of all issues resolved in this release, see the list of link:https://issues.redhat.com/browse/WINDUP-2759?filter=12357835[{ProductVersion} resolved issues] in Jira.
```
Comment (apinnick) [inline docs/topics_5/rn-resolved-issues.adoc:8]: Use this attribute

**Comment by PhilipCattanach** [comment]
Thanks @RichardHoch the preview looks good.

**Comment by RichardHoch** [comment]
@apinnick -- fixed the attributes. Please review.

File: docs/topics_5/rn-resolved-issues.adoc [hunk 5]
```diff
@@ -7,23 +7,23 @@ At the time of the release the following resolved issues have been identified as
...
-|The MTA web console installed on OpenShift by means of a template cannot access the reports using HTTPS.
+|The {WebName} installed on OpenShift 4.4 and earlier versions cannot access the generated reports over an HTTPS connection.
 
 |link:https://issues.redhat.com/browse/WINDUP-2801[WINDUP-2801]
-|The MTA web console installed on OpenShift by means of the MTA Operator cannot access the reports using HTTPS.
+|The {WebName} installed on OpenShift 4.5 and later versions cannot access the generated reports over an HTTPS connection.
 
 |link:https://issues.redhat.com/browse/WINDUP-2910[WINDUP-2910]
-|"Unparseable Class File" errors are generated when MTA analyzes applications built with JDK 11.
+|Analysis of applications built with JDK 11 generates `"Unparseable Class File"` errors.
```
Comment (apinnick) [inline docs/topics_5/rn-resolved-issues.adoc:26]: You should remove the quotes. Just the backtick markup should be around the error message.

**Comment by RichardHoch** [comment]
@apinnick -- Name of table 4.1 changed; commits squashed. Please review and merge.

**Comment by RichardHoch** [comment]
@apinnick: changed name of table 4.1, removed quotations around error named in the table, and squashed my commits. Please review and merge.

**Pull request merged**